demo/AliceGetsAPhone.md

```diff
@@ -118,25 +120,41 @@ ngrok-tails-server_1  | t=2020-05-13T22:51:14+0000 lvl=info msg="started tunnel"
 
 Note the server name in the `url=https://c5789aa0.ngrok.io` parameter (`https://c5789aa0.ngrok.io`) - this is the external url for your tails server. Make sure you use the `https` url!
 
+#### Running in Play with Docker?
+
+Run the same steps on _PWD_ as you would run locally (see above).  Open a new shell (click on "ADD NEW INSTANCE") to run the tails server.
+
+Note that with _Play with Docker_ it can be challenging to capture the information you need from the log file as it scrolls by, you can try leaving off the `--events` option when you run the Faber agent to reduce the quantity of information logged to the screen.
+
 ### Run `faber` With Extra Parameters
 
-If you are running in a _local bash shell_, navigate to [The demo direcory](/demo) and run:
+#### Running locally in a bash shell?
+
+If you are running in a _local bash shell_, navigate to [The demo directory](/demo) and run:
 
 ```bash
 TAILS_NETWORK=docker_tails-server LEDGER_URL=http://test.bcovrin.vonx.io ./run_demo faber --revocation --events
 ```
 
 The `TAILS_NETWORK` parameter lets the demo script know how to connect to the tails server (which should be running in a separate shell on the same machine).
 
-If you are running in _Play with Docker_, navigate to [The demo direcory](/demo) and run:
+#### Running in Play with Docker?
+
+If you are running in _Play with Docker_, navigate to [The demo directory](/demo) and run:
 
 ```bash
-PUBLIC_TAILS_URL=https://def456.ngrok.io LEDGER_URL=http://test.bcovrin.vonx.io ./run_demo faber --revocation --events
+PUBLIC_TAILS_URL=https://c4f7fbb85911.ngrok.io LEDGER_URL=http://test.bcovrin.vonx.io ./run_demo faber --revocation --events
```

Comment:
So they still have to run ngrok because of the https requirement, right?  I had been thinking that the public tails file would be local.  Can the run ngrok on PWD?

Also -- I assume that they need to replace the ngrok subdomain with the one they are using, right?  I don't see anything to tell them that.  Obvious, but....  :-)

Reply:
When you run the tails server docker-compose it starts ngrok automatically, so it's the same running local or on PWD

I gave it a test this afternoon